consensus/src/round_manager_fuzzing.rs

```diff
@@ -168,21 +158,16 @@ fn create_node_for_fuzzing() -> RoundManager {
     )
 }
 
-// This functions fuzzes a Proposal protobuffer (not a ConsensusMsg)
-pub fn fuzz_proposal(data: &[u8]) {
+//
+// Proposal Fuzzing
+//
+
+/// Fuzz a proposal message
+pub fn fuzz_proposal(proposal: ProposalMsg) {
```

Comment:
hey @zekun000 any chance you can review this part :) ? looking to improve the fuzzer here and see what we can mock better

Reply:
(I suspect that the `create_node_for_fuzzing` can be improved greatly)

Reply:
why we can't use the  NodeSetup::create_nodes for this? do I miss anything?